frontend/components/bridge/utils.ts

```diff
@@ -41,19 +47,86 @@ const getFromToken = (
   return fromChainConfig[tokenSymbol].address as Address;
 };
 
+/**
+ *
+ * @warning A HOOK THAT SHOULD NEVER EXIST.
+ * @deprecated TODO: This hook is planned for removal in future version
+ *
+ * Hook to return the updated bridge requirements params to improve the
+ * initial funding requirements.
+ *
+ * Request quote with formula (will be moved to backend):
+ *   max(refill_requirement_masterSafe, monthly_gas_estimate) + refill_requirements_masterEOA
+ *
+ */
+const useGetUpdatedBridgeRequirementsParams = () => {
```

Comment:
can we rename to something like:
```suggestion
const useGetBridgeRequirementsParamsWithMonthlyGasEstimate = () => {
```

or even

```suggestion
const useGetBridgeParamsWithMonthlyGasEstimate = () => {
```